### PR TITLE
Move example solution to .meta

### PR DESCRIPTION
**Pull request by angelikatyborska**

### CODE REVIEW

**Comment by angelikatyborska** [comment]
@ErikSchierboom I believe the fix to hello-world (missing config) is necessary before launch

**Comment by angelikatyborska** [comment]
Oops, didn't notice that there was an open PR for this already... https://github.com/exercism/lfe/pull/113

File: exercises/practice/hello-world/.meta/config.json [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-  "blurb": "The classical introductory exercise. Just say \"Hello, World!\"",
+  "blurb": "The classical introductory exercise. Just say \"Hello, World!\".",
   "authors": [
     "benreyn"
   ],
-  "contributors": [
-    "agentofuser",
-    "NobbZ",
-    "yurrriq"
-  ],
```
Comment (ErikSchierboom) [inline exercises/practice/hello-world/.meta/config.json:10]: What was the reason for removing these contributors?

**Review by ErikSchierboom — APPROVED** [review]
One small question

File: exercises/practice/hello-world/.meta/config.json (hunk 1 above)
Comment (angelikatyborska) [reply exercises/practice/hello-world/.meta/config.json:10]: Hmm. I must have messed up something git-wise. On my branch, hello-world was missing a few files. As you can see in this commit [`4a5d6a0` (#139)](https://github.com/exercism/lfe/pull/139/commits/4a5d6a07a81816da93bcca8f4379d95022e55196) it adds them, not edits them.

I was convinced those files are missing on main too. Let me fix this.

Comment (ErikSchierboom) [reply exercises/practice/hello-world/.meta/config.json:10]: Sure

**Comment by angelikatyborska** [comment]
I just purged any hello-world changes other than moving the example. I don't know what happened... now this PR is just a copy of https://github.com/exercism/lfe/pull/113/files, but without conflicts 🙈

**Comment by ErikSchierboom** [comment]
@angelikatyborska Thanks!

**Pull request merged**